index.html

```diff
@@ -141,7 +130,7 @@ <h1 class="work-title">Multi-Post stories</h1>
                         <img src="img/work-3.png" alt="tonic" class="work-img_c">
                     </div>
                     <div class="work-content">
-                        <h1 class="work-title">Tonic</h1>
+                        <h1 class="work-title ">Tonic</h1>
                         <ul class="work-chronical">
                             <li class="work-chronical-title">CANOPY</li>
                             <li class="work-chronical_c">Back End Dev</li>
```

Comment:
- [ ] The text of the ul/il should be bigger than the project description
![image](https://user-images.githubusercontent.com/62705947/177798639-bff563e5-faa4-4158-a49a-a35698af6264.png)